docker/sakai/placeholder.properties

```diff
@@ -275,6 +275,12 @@ calendar.export.previous.months = 6
 # Number of months in the future required in the export
 calendar.export.next.months = 12
 
+# Start the public announcement system
+pasystem.enabled=true
+pasystem.auto.ddl=true
```

Comment:
Normally we don't do DDL so that we know what's changing regarding the SQL with each upgrade and can test it. I think the PA system tables are already created as part of the Sakai 10 -> 11 upgrade.